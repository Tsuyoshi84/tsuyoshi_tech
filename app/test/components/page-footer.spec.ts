import { mount, RouterLinkStub } from '@vue/test-utils'
import PageFooter from '@/components/page-footer.vue'

const makeWrapper = () => {
  return mount(PageFooter, {
    stubs: {
      NuxtLink: RouterLinkStub,
    },
  })
}

describe('PageFooter', () => {
  test('is a Vue instance', () => {
    const wrapper = makeWrapper()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('generate DOM', () => {
    const wrapper = makeWrapper()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
