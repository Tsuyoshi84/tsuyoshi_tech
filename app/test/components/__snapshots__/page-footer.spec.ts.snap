// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter generate DOM 1`] = `<footer><span>Using <a href="https://fontawesome.com/license" alt="Font Awesome">Font Awesome </a></span></footer>`;
